configure.py

```diff
@@ -1392,6 +1395,9 @@ def query_seastar_flags(seastar_pc_file, link_static_cxx=False):
         f.write('rule libdeflate.{mode}\n'.format(**locals()))
         f.write('    command = make -C libdeflate BUILD_DIR=../build/{mode}/libdeflate/ CFLAGS="{libdeflate_cflags}" CC={args.cc} ../build/{mode}/libdeflate//libdeflate.a\n'.format(**locals()))
         f.write('build build/{mode}/libdeflate/libdeflate.a: libdeflate.{mode}\n'.format(**locals()))
+        f.write('rule zstd.{mode}\n'.format(**locals()))
+        f.write('    command = make -C zstd/lib BUILD_DIR=../../build/{mode}/zstd/ CFLAGS="{zstd_cflags}" CC={args.cc} ../../build/{mode}/zstd//libzstd.a\n'.format(**locals()))
```

Comment:
What's this `../../build`? Isn't that the same dir we are in now?

Reply:
Suppose the working directory is Scylla's main directory.
Zstd's Makefile lies in ./zstd/lib.
We tell the Makefile to build the "../../build/dev/zstd//libzstd.a" target (when mode=dev), which is specified here:
https://github.com/kbr-/zstd/blob/1.4.0/lib/Makefile#L152
in this case $(BUILD_DIR) = "../../build/dev/zstd/".
"../../build/dev/zstd//libzstd.a" where PWD="./zstd/lib" resolves to to ./build/dev/zstd/ibzstd.a.

Reply:
I see.